cli/emit.rs

```diff
@@ -368,7 +368,7 @@ pub(crate) fn check_and_maybe_emit(
       let root_str = root.to_string();
       // `$deno` specifiers are internal, don't print them.
       if !root_str.contains("$deno") {
-        log::info!("{} {}", colors::green("Check"), root);
+        eye_catchers::check(&format!("{}", root));
```

Comment:
Previously if someone had log level set to not do anything for info then it wouldn't allocate a string here. I don't think that really matters though because these are "info" logs and not something like "debug" or "trace". That said, I wonder if it would be better to make all these macros anyway? Then we wouldn't have to use a `format!` macro. Thoughts @bartlomieju?

Reply:
I don't think it's a big deal, that said, it seems `format!` is not needed here anyway

Reply:
Yeah, it's not needed here. I've removed it already.